angular/projects/spark-angular/src/lib/components/sprk-award/sprk-award.component.ts

```diff
@@ -14,7 +14,7 @@ import { Component, Input } from '@angular/core';
           sprk-o-Stack__item
           sprk-b-TypeDisplayFive
           sprk-b-Measure sprk-b-Measure--narrow
-          sprk-u-TextAlign--center
+          sprk-b-Type--center
```

Comment:
Added these new type modifiers to type

Reply:
Why would we make a new class just for this? I see that we're moving away from utility classes, but this effectively does the same thing. Is it because we don't have a component class on this?

I know a lot of consideration went into this! I do have some first draft thoughts:

- Are we no longer using text-align center in any of our component classes because we can use this class? I do see the value in having a class without `!important` in it, but it does feel redundant.
- Are we putting ourselves into maintenance/hidden context prison? Multiple classes doing the same thing. I feel like a new developer would need to do detective work `b-Type--center class` to know this exists unless we tell them.
- Rather than creating a new base class, could this possibly be a `Stack__item` modifier instead?
- I like atomic styles, but I know our design system doesn't really do it. This feels like a step toward a direction we've moved away from.

Reply:
@RVMendoza Good questions,  we are moving away from using !important because of feedback. People can't override styles. So instead, we are moving some classes to be modifiers and keeping some as utils. People want the ability to adjust styles via the atomic style classes such as this one for type. So it's a modifier for type. We will probably add even more type modifiers as we go. We are going to look at the util classes and remove certain ones.